src/java.base/share/classes/java/lang/ScopedValue.java

```diff
@@ -311,7 +309,6 @@ Object find(ScopedValue<?> key) {
      *
      * @since 21
      */
-    @PreviewFeature(feature = PreviewFeature.Feature.SCOPED_VALUES)
```

Comment:
The @since need to be updated for all APIs that are proposed to be permanent. This means that Carrier and CallableOp need their @since updated.